azurerm/resource_arm_api_management_api_policy.go

```diff
@@ -74,14 +74,14 @@ func resourceArmApiManagementAPIPolicyCreateUpdate(d *schema.ResourceData, meta
 
 	if xmlContent != "" {
 		parameters.PolicyContractProperties = &apimanagement.PolicyContractProperties{
-			ContentFormat: apimanagement.XML,
+			ContentFormat: apimanagement.Rawxml,
 			PolicyContent: utils.String(xmlContent),
 		}
 	}
 
 	if xmlLink != "" {
 		parameters.PolicyContractProperties = &apimanagement.PolicyContractProperties{
-			ContentFormat: apimanagement.XMLLink,
+			ContentFormat: apimanagement.Rawxml,
```

Comment:
since this downloads the XML from the specified link, I have a feeling this needs to be:

```suggestion
			ContentFormat: apimanagement.RawxmlLink,
```

which should still allow this to be treated as RawXML, but ensure it's downloaded rather than interpreted as XML directly